src/host.rs

```diff
@@ -24,9 +24,52 @@ pub(crate) enum HostInternal {
     Ipv6(Ipv6Addr),
 }
 
-impl<S> From<Host<S>> for HostInternal {
+#[cfg(feature = "serde")]
```

Comment:
Why is this back?

Reply:
https://github.com/servo/rust-url/commit/171e65e5c982480fc81b428d44228f153435d0a8 I didnt notice that, sorry ^^